charts/pxc-operator/Chart.yaml

```diff
@@ -1,9 +1,9 @@
 apiVersion: v1
 appVersion: "1.3.0"
-description: A Helm chart for Deploying the Percona XtraDB Cluster Operator Kubernetes
+description: A Helm 3 chart for Deploying the Percona XtraDB Cluster Operator Kubernetes
```

Comment:
I don't think we need to specify `Helm 3` in the description.  This chart should continue to work for `Helm 2` as well, with the additional instructions of `kubectl apply -f crd/` beforehand.

Reply:
ok